mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(amount) {
+  result = ((20/100*amount) + amount).toFixed(2);
```

Comment:
Your solution works fine apart from the reference error I mentioned above. However, as you already have a function above that calculates sales tax above, you could just call that function instead of doing the calculation again. Something like:
`let result = calculateSalesTax(amount).toFixed(2);`